dev/AppAttach/AppAttachFramework/AppAttachAPI/Exceptions/Messages.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Reflection;
+using System.Xml;
+
+namespace AppAttachAPI.Exceptions
+{
+    /// <summary>
+    /// A static class containing error messages for various publishing-related errors.
+    /// </summary>
+    public static class Messages
+    {
+        private static Hashtable _errorTable;
+
+        // TODO: Add support for localization
+        /*
+         * private static CultureInfo _cultureInfo = CultureInfo.InstalledUICulture;
+         * private static readonly string LOCALE = _cultureInfo.Name;
+        */
```

Comment:
DELETE: Don't include dead code. If this is part of a localization solution then mention it in the ADO item to add localization; don't assume what the future implementation will look like